### PR TITLE
Init

### PR DESCRIPTION
**Pull request by RomeoKs**
- [DEMO LINK](https://RomeoKs.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://RomeoKs.github.io/layout_product-cards-en/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,82 @@
...
+    >
+  </head>
+
+  <body>
+    <div
+      class="card"
+      data-qa="card"
+    >
+
+      <img
```
Comment (mgrinko) [inline src/index.html:35]: this line is redundant
div and img are not siblings

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,82 @@
...
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+
+      <div class="card__review-wrapper">
+
+        <div class="stars stars--4">
```
Comment (mgrinko) [inline src/index.html:51]: empty line only between siblings
Not the parent and its child

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  min-height: 100vh;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```
Comment (mgrinko) [inline src/style.css:19]: it should be the first rule because it changes box model

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  margin: 0;
+}
+
+/* #region Product Code */
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary);
+  margin: 4px 0 16px;
```
Comment (mgrinko) [inline src/style.css:43]: don't use both mt and mb (choose one) for siblings to awoid possible margin collapse
I prefer `mt`

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__review-text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-accent);
```
Comment (mgrinko) [inline src/style.css:85]: You could make this font the default one for the page and avoid writing these rules twice

**Review by mgrinko — APPROVED** [review]
Take into account in your next solutions

**Comment by RomeoKs** [comment]
> Take into account in your next solutions

I will do my best and thank you!

**Pull request closed without merging**